src/AzureFunctions/Example/StringExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using GraphQL;
+using GraphQL.NewtonsoftJson;
+using Newtonsoft.Json.Linq;
+
+public static class StringExtensions
```

Comment:
Is this file definitely needed?

Reply:
I would not think that it's needed.  Feel free to remove it in a subsequent PR along with changing any code that referenced it.